app/src/main/java/ch/epfl/sdp/map/PathsHandler.java

```diff
@@ -193,6 +196,32 @@ public void setCameraPosition(int day) {
         }
     }
 
+    public void seeWholePath(int day) {
+        Log.d("SEEWHOLEPATH: ", "enter method");
+        LatLngBounds latLngBounds = day == R.string.yesterday ? yesterdayLLB : beforeLLB;
+
+        if (map != null) {
+            Log.d("SEEWHOLEPATH: ", "new lat lng bounds");
+            map.easeCamera(CameraUpdateFactory.newLatLngBounds(latLngBounds, 100), 2000);
+        }
+    }
+
+    private LatLngBounds setLatLngBounds(int day) {
+        List<Point> pathCoord = day == R.string.yesterday ? yesterdayPathCoordinates :
+                beforeYesterdayPathCoordinates;
+        Log.d("SETLATLNGBNDS: ", "pathCoord is empty? " + pathCoord.isEmpty());
+        LatLngBounds.Builder boundsBuilder = new LatLngBounds.Builder();
+        double lat;
```

Comment:
Is `lat` and `lng` relevant outside of the scope of the for?

Reply:
Thanks for this remark, I can put them inside the for